src/cobalt_model/pagination_config.rs

```diff
@@ -156,6 +179,7 @@ impl Default for PaginationConfig {
             front_permalink: Default::default(),
             order: SortOrder::Desc,
             sort_by: vec![DEFAULT_SORT.to_owned()],
+            date_index: DEFAULT_DATE_INDEX.to_vec(), // TODO default value to define
```

Comment:
Just creating this to track the TODO

Reply:
Oh, this TODO was to confirmed the value I chose: `Year, Month` as default. If that's ok with you, I'll remove the TODO

Reply:
Thats a good default, so feel free to remove the TODO.